CrrExistingObjects/metadataUtils.js

```diff
@@ -85,26 +91,33 @@ function _getNullVersion(objMD, bucketName, objectKey, log, cb) {
 }
 
 function getMetadata(params, log, cb) {
+    const { Bucket, Key } = params;
```

Comment:
I think these two parameters should be lower case, eg `bucket` and `key`

Reply:
It was made/kept because the original requests to s3 client expected them. Also at the time of making the original improvement (which is to use mongodbWrapper instead of cloudserver and backbeat) there was a discussion to have both options and switch based on user option. We may revisit this and hence to avoid breaking changes, it was intentionally left as it is.